docs/DeveloperGuide.adoc

```diff
@@ -48,6 +48,7 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 |`* *` |user |hide <<private-contact-detail, private contact details>> by default |minimize chance of someone else seeing them by accident
 |`*` |user with many persons in the address book |sort persons by name |locate a person easily
+|`* * *` |meticulous user |add notes along with my contacts | keep track of my working relations with colleagues
```

Comment:
The user story added is fine. If possible try to arrange the user stories such that the ones with 3 stars are on top, followed by those with 2 stars and finally those with 1 star.

Reply:
You can close the PR once you are done reviewing.

Reply:
Noted the suggestions. Thank you!